src/mlpack/core/cv/metrics/r2_score.hpp

```diff
@@ -53,12 +53,15 @@ class R2Score
    * @param data Column-major data containing test items.
    * @param responses Ground truth (correct) target values for the test items,
    *     should be either a row vector or a column-major matrix.
+   * @param adjR2 Boolean value which specifies whether to calculate adjusted
+   *    R2 or not
    * @return calculated R2 Score.
    */
   template<typename MLAlgorithm, typename DataType, typename ResponsesType>
   static double Evaluate(MLAlgorithm& model,
                          const DataType& data,
-                         const ResponsesType& responses);
+                         const ResponsesType& responses,
+                         const bool adjR2 = false);
```

Comment:
I think we can be more explicit in the naming, what about `adjustedRSquared`?

Reply:
I had actually thought of this and I came to the conclusion that it was just a long argument name. So I shortened it. But if you want rename it to `adjustedRSquared`, please do tell.